apps/address-service/domains/common/utils/services/suggest/SuggestionKeystoneApp.js

```diff
@@ -123,6 +123,9 @@ class SuggestionKeystoneApp {
                     count,
                     helpers,
                 })
+
+                // TODO(AleX83Xpert): Think about splitting the address string to tokens, compile 2nd variant of address and pass to suggestion provider
```

Comment:
Good practice for leaving todo note is connect it to concrete task

Reply:
Done